be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -132,15 +133,14 @@ Status JsonScanner::_construct_json_types() {
                 slot_type->type == TYPE_TINYINT) {
                 // Treat these types as what they are.
                 child_type->children.emplace_back(slot_type->type);
-
             } else if (slot_type->type == TYPE_VARCHAR) {
                 auto varchar_type = TypeDescriptor::create_varchar_type(slot_type->len);
                 child_type->children.emplace_back(varchar_type);
-
             } else if (slot_type->type == TYPE_CHAR) {
                 auto char_type = TypeDescriptor::create_char_type(slot_type->len);
                 child_type->children.emplace_back(char_type);
-
+            } else if (slot_type->type == TYPE_JSON) {
```

Comment:
It's better to add UT case to cover this.

Reply:
Fixed